lib/pstypes.h

```diff
@@ -26,7 +26,7 @@ typedef unsigned char ubyte;
 typedef signed char sbyte;
 typedef unsigned short ushort;
 typedef unsigned int uint;
-typedef unsigned long ulong;
+typedef unsigned int ulong;
```

Comment:
This cause compilation failures. Need to revert back to long.

Reply:
I'm not sure that's the problem. Converting `unsigned long` to `unsigned int` means LPDWORDs need static casts.